databricks-example/docs/setup-dbfs.md

```diff
@@ -0,0 +1,51 @@
+# Set up Databricks Feature Store
+
+## Setup Azure Databricks instance
+
+There are two main steps involved when creating a Databricks cluster:
+  1. Creating the databricks infrastructure on Azure portal and
+  2. Creating the databricks cluster itself.
+Detailed below is the configurations settings and setup required for each the above steps.
+
+Creation of the resources on Azure:
+
+![databricks-workspace-settings.png](./images/databricks-workspace-settings.png)
+
+Creation of the cluster on Databricks:
+
+![databricks-cluster-settings.png](./images/databricks-cluster-settings.png)
+
+Important to note that we are using the 10.2 ML Databricks Runtime. This contains APIs that allow us to use the built-in feature store relevant functionality.
+
+Azure Databricks comes with Feature Store capabilities out of the box. Upon navigating to the Machine learning section on the left menu, one should be able ot view the 'Feature Store' section.
+
+![az-db-fs.png](./images/az-db-fs.png)
+
+## Create sample data, feature views & features on Azure Databricks Feature Store
```

Comment:
Is this section relevant as it uses the NYC Taxi dataset?

Reply:
Redacted the NYC data on the screenshot